drivers/misc/cldatadrv/r_taurus_protocol_ids.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _R_TAURUS_PROTOCOL_IDS_H_
```

Comment:
We already have the same file in `drivers/gpu/drm/rcar-rvgc/`. No need to make copies. Just move the original file to some shared location